dev/compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  solar-protocol:
+    image: solar-protocol
+    restart: always
+    ports:
+      - 80
```

Comment:
On my machine apache2 default port ends up being 43753. We should specify a known port to expose locally to be extra sure.